### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by vritant24**
Made changes to the UI as specified by the project owner
Navbar and Logo not included

### CODE REVIEW

File: public/graph.js [hunk 1]
```diff
@@ -131,18 +134,18 @@ var initGraph = function() {
...
-    graphOne();
-    if (currentPondVolume) {
-      graphTwo();
-      if (year) {
-        graphThree();
-      }
-    }
-  }
-});
+// //Resizes Graph on window resize
```
Comment (drewatk) [inline public/graph.js]: If this doesn't need to be kept, I'd remove it

File: views/partials/header.ejs [hunk 2]
```diff
@@ -1,7 +1,10 @@
 <nav class="navbar">
   <div class="container-fluid">
     <div class="navbar-header">
-      <a class="navbar-brand" href="/">Reservoir Planning Tool</a>
+      <a class="navbar-brand" href="/">
+        <!-- <img alt="Brand" class="img-responsive" src="tdp-logo.png"> -->
```
Comment (drewatk) [inline views/partials/header.ejs]: Did the image not fit in the navbar?

Comment (vritant24) [inline views/partials/header.ejs]: It didn't

File: public/graph.js (hunk 1 above)
Comment (vritant24) [inline public/graph.js]: Good catch

File: views/partials/header.ejs (hunk 2 above)
Comment (drewatk) [inline views/partials/header.ejs]: Can we remove the file if we're not using it?

**Pull request merged**